scripts/linters/html_linter.py

```diff
@@ -419,3 +419,23 @@ def perform_all_lint_checks(self):
             return []
 
         return self._lint_html_files()
+
+
+def get_linters(files_to_lint, verbose_mode_enabled=False):
+    """Creates HTMLLintChecksManager and ThirdPartyHTMLLintChecksManager
+        objects and returns them.
+
+    Args:
+        files_to_lint: list(str). A list of filepaths to lint.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
+
+    Returns:
+        linter: object(linter). Returns linter object.
+    """
+    custom_linter = HTMLLintChecksManager(
+        files_to_lint, verbose_mode_enabled)
+
+    third_party_linter = ThirdPartyHTMLLintChecksManager(
+        files_to_lint, verbose_mode_enabled)
+
+    return custom_linter, third_party_linter
```

Comment:
Here you return two things, elsewhere you return one. The returns docstring typeinfo for this function is also wrong.

Please standardize:
- If you need to draw a distinction between custom and third-party linters, then have all these methods return a 2-tuple (perhaps with one of the values being None) -- the first value is always the custom linter, the second is always the third-party linter.
- If you don't need that sharp distinction, then just have this function return a list of linters in each case, where the list may be of arbitrary length. That way the main orchestrating function does not need to know the details which files return one linter and which return two.

Reply:
Used tuple.